sdk/src/types/filters.ts

```diff
@@ -58,6 +58,7 @@ export type FilterOperators<
 	_nbetween: IsDateTime<FieldType, [T, T], IsNumber<T, [T, T], never>>;
 	_contains: IsDateTime<FieldType, never, IsString<T, string, never>>;
 	_ncontains: IsDateTime<FieldType, never, IsString<T, string, never>>;
+	_icontains: IsDateTime<FieldType, never, IsString<T, string, never>>;
```

Comment:
Should we add the `_nicontains` operator?

```suggestion
	_icontains: IsDateTime<FieldType, never, IsString<T, string, never>>;
	_nicontains: IsDateTime<FieldType, never, IsString<T, string, never>>;
```

Reply:
Currently `_nicontains` does not pass the filter validation this does not work on the API.
![image](https://github.com/directus/directus/assets/9389634/0493de15-b876-4211-9563-9e364a5f1f0e)

But good point lets pick that up in a separate issue.

Reply:
Was wondering the same 🤔 Though this one is not documented, right? So probably better to addressed that separately, including docs and potentially other implications.